### ISSUE TITLE
recipe for finetuning OneFormer

### ISSUE THREAD

**Issue by werner-rammer** [NielsRogge/Transformers-Tutorials#330]
For Mask2Former there is a fantastic tutorial for finetuning (e.g., https://github.com/NielsRogge/Transformers-Tutorials/blob/master/MaskFormer/Fine-tuning/Fine_tuning_MaskFormer_on_a_panoptic_dataset.ipynb).

It would be very useful to have a similar tutorial for OneFormer. Specific questions are:
- how to provide own classes / labels?
- How to use OneFormerProcessor with own data?

A step by step tutorial such as the one mentioned above would be very welcome!

**Comment by nickponline** [comment]
I'm also looking for an example. The text said it was the same as MaskFormer and Mask2Former by in reality it seems very different.

**Comment by NielsRogge** [comment]
A notebook has now been uploaded here: https://github.com/NielsRogge/Transformers-Tutorials/blob/master/OneFormer/Fine_tune_OneFormer_for_semantic_segmentation.ipynb.
